pkgs/tools/misc/expect/default.nix

```diff
@@ -9,11 +9,12 @@ tcl.mkTclDerivation rec {
     hash = "sha256-Safag7C92fRtBKBN7sGcd2e7mjI+QMR4H4nK92C5LDQ=";
   };
 
-  patches = [
+  patches = lib.optionals (!stdenv.isFreeBSD) [
     (fetchpatch {
       url = "https://raw.githubusercontent.com/buildroot/buildroot/c05e6aa361a4049eabd8b21eb64a34899ef83fc7/package/expect/0001-enable-cross-compilation.patch";
       hash = "sha256-yyzE0Jjac5qaj7Svn4VpMiAqSNLYrw7VZbtFqgMVncs=";
     })
+  ] ++ [
```

Comment:
Generally, conditionally-applied patches are bad news, but looking at the contents of this patch, it also looks like bad news, so I guess it's fine but I wonder if there's a better way we could do this…

If we are to make it conditional, could it at least be moved after the unconditional ones?